docs/strategy-customization.md

```diff
@@ -249,6 +249,22 @@ minimal_roi = {
 
 While technically not completely disabled, this would sell once the trade reaches 10000% Profit.
 
+To use times based on candles, the following snippet can be handy.
+This will allow you to change the ticket_interval, and ROI will be set as candles (e.g. after 3 candles ...)
+
+``` python
+from freqtrade.exchange import timeframe_to_minutes
+
+class AwesomeStrategy(IStrategy):
+
+    ticker_interval = '1d'
+    ticker_interval_mins = timeframe_to_minutes(ticker_interval)
+    minimal_roi = {
+        (ticker_interval_mins * 3): 0.02,  # After 3 candles
```

Comment:
Has this been tested? It does not seem to work for me, also shouldn't it start with 0 minutes?

Reply:
Yes i did - the 0 entry is not necessary - however there was a bug in this section - it needs explicit conversion to string as this is expected to be a string.

A sample configuration (it did a few trades in this config, too)

![2020-02-26-064243_492x258_scrot](https://user-images.githubusercontent.com/5024695/75315621-7e2b6080-5863-11ea-99e2-a1ca0cc8552b.png)